person.rb

```diff
@@ -0,0 +1,31 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # accessor get method
+  attr_reader :id
+
+  # accessor get and set method
+  attr_accessor :name, :age
+
+  # Public method
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
+  end
+
+  # private method
+
+  private
+
+  def of_age?
+    return true if age >= 18
+
+    false
```

Comment:
- [Optional] The private method `of_age?` will always return true if age is greater or equal to 18 and false otherwise. This can be simply written as :

```ruby
  def of_age?
     age >= 18
  end
```

Reply:
Hello @FrederickMih, I'm new to ruby syntaxes and this tips help me a lot. Many thanks ❤️